cmd/lk/agent_dispatch.go

```diff
@@ -0,0 +1,209 @@
+// Copyright 2023 LiveKit, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/urfave/cli/v3"
+
+	"github.com/livekit/protocol/livekit"
+	lksdk "github.com/livekit/server-sdk-go/v2"
+)
+
+var (
+	AgentDispatchCommands = []*cli.Command{
+		{
+			Name:     "agentdispatch",
```

Comment:
We should probably discuss the proper name/command hierarchy here, since `agentdispatch` may not be clear

Reply:
I see nothing wrong with a `dispatch` subcommand of `agent` command:

```
lk agent dispatch create --room my_room --agent-name bot
```